.github/workflows/save-api-examples-release.yml

```diff
@@ -0,0 +1,53 @@
+name: Create release
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'save-api/**'
+      - 'save-api-cli/**'
+env:
+  GPG_SEC: ${{ secrets.PGP_SEC }}
+  GPG_PASSWORD: ${{ secrets.PGP_PASSWORD }}
+  OSSRH_USERNAME: ${{ secrets.SONATYPE_USER }}
+  OSSRH_PASSWORD: ${{ secrets.SONATYPE_PASSWORD }}
+
+jobs:
+  release:
+    name: Build release
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ ubuntu-latest, windows-latest, macos-latest ]
```

Comment:
It's a jvm-only artifact, I think we don't need a matrix build

Reply:
Fixed